contracts/system-contracts/hedera-token-service/IHTSStructs.sol

```diff
@@ -0,0 +1,316 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.4.9 <0.9.0;
+
+interface IHTSStructs {
+    /// Transfers cryptocurrency among two or more accounts by making the desired adjustments to their
+    /// balances. Each transfer list can specify up to 10 adjustments. Each negative amount is withdrawn
+    /// from the corresponding account (a sender), and each positive one is added to the corresponding
+    /// account (a receiver). The amounts list must sum to zero. Each amount is a number of tinybars
+    /// (there are 100,000,000 tinybars in one hbar).  If any sender account fails to have sufficient
+    /// hbars, then the entire transaction fails, and none of those transfers occur, though the
+    /// transaction fee is still charged. This transaction must be signed by the keys for all the sending
+    /// accounts, and for any receiving accounts that have receiverSigRequired == true. The signatures
+    /// are in the same order as the accounts, skipping those accounts that don't need a signature.
+    /// @custom:version 0.3.0 previous version did not include isApproval
+    struct AccountAmount {
```

Comment:
Is it possible to have top-level structs? So to avoid extending the interface just to use some struct.

Reply:
Good question, i'm not sure. My goal was by having this other interfaces wouldn't have to re-specify I didn't want to do too much here but eventually the system contract files could inherit that and have the structs accessible

Reply:
Got it.